feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: dpdk-network-node-policy-dpdk
```

Comment:
Can this be either `dpdk-network-node-policy` or `network-node-policy-dpdk` ?

Reply:
ops yep my bad nice catch